### PR TITLE
[docs] Update `CONTRIBUTING.md`, add pull request templates

### PR DESCRIPTION
**Pull request by tsmethurst**
Update contributing.md:

- Add pull request processes. These just codify what we already did anyway.
- Add guidelines for opening feature requests + bug reports.
- Move stub CoC to its own separate document, where it can more easily be updated later.
- Add a section to help new contributors find their way around the code.

Add pull request templates, which reference the updated contributing.md.

### CODE REVIEW

**Comment by mal0ki** [comment]
I have a TON of comments for other things than the actual changes here, so I will try to handle that elsewhere.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Code of Conduct
+
+This document is currently a stub.
+
+In lieu of a fuller code of conduct, here are a few ground rules.
+
+1. We *DO NOT ACCEPT* PRs from right-wingers, Nazis, transphobes, homophobes, racists, harassers, abusers, white-supremacists, misogynists, tech-bros of questionable ethics. If that's you, politely fuck off somewhere else.
```
Comment (NyaaaWhatsUpDoc) [inline CODE_OF_CONDUCT.md:7]: Is it worth including ableists here? And some means of including "enlightened centrist" types of any kind, i.e. anyone who hates the "left" and "SJWs"?

Comment (NyaaaWhatsUpDoc) [reply CODE_OF_CONDUCT.md:7]: Also:
- absolutely no capitalism supporters
- no cop supporters

Comment (tsmethurst) [reply CODE_OF_CONDUCT.md:7]: Both good ideas :+1: For now this PR is mostly about the contributing.md document. The CoC here is just copied from what we had previously as a stub, so I think we can work on it outside this PR

**Review by f0x52 — APPROVED** [review]

**Pull request merged**